eng/containers/UbuntuNetCore2Keyring/Dockerfile

```diff
@@ -1,6 +1,46 @@
-FROM mcr.microsoft.com/dotnet/core/sdk:2.1 AS build
+FROM mcr.microsoft.com/dotnet/core/sdk:2.1-bionic AS build
 
-RUN apt-get update && apt-get install -y libsecret-1-dev dbus-x11 gnome-keyring
+ENV \
+    NO_AT_BRIDGE=1 \
+    DOCKER_CONTAINER_NAME="ubuntu_netcore2_keyring" \
+    PSModuleAnalysisCachePath=/var/cache/microsoft/powershell/PSModuleAnalysisCache/ModuleAnalysisCache \
+    POWERSHELL_DISTRIBUTION_CHANNEL=PSDocker-DotnetCoreSDK-Ubuntu-18.04
 
-ENV NO_AT_BRIDGE=1
-ENV DOCKER_CONTAINER_NAME=".Net Core 2.1 with Keyring"
+# Install GNOME keyring
+RUN apt-get update \
+    && apt-get install -y \
+        libsecret-1-dev \
+        dbus-x11 \
+        gnome-keyring
+
+# Install PowerShell
+RUN apt-get update \
+    && apt-get install --no-install-recommends -y \
+        locales \
+        ca-certificates \
+        gss-ntlmssp \
+        openssh-client
+
+RUN powershell_version=6.2.0 \
```

Comment:
Why version 6.2? As opposed to the latest version? In case it is interesting https://github.com/actions/virtual-environments/blob/main/images/linux/scripts/installers/powershellcore.sh that is the script that DevOps uses to install pwsh .